pkg/kubelet/metrics/collectors/volume_stats.go

```diff
@@ -61,6 +61,12 @@ var (
 		[]string{"namespace", "persistentvolumeclaim"}, nil,
 		metrics.ALPHA, "",
 	)
+
+	volumeStatsHealthAbnormalDesc = metrics.NewDesc(
+		metrics.BuildFQName("", kubeletmetrics.KubeletSubsystem, kubeletmetrics.VolumeStatsHealthStatusKey),
+		"Volume health status. The count is either 1 or 0. When volume_health_status is abnormal, 1 indicates the volume is unhealthy, 0 indicates volume is healthy",
```

Comment:
Should we change "Volume health status" to "Abnormal volume health status" or "Volume health status - Abnormal"?

Reply:
Agree, `Volume health status: 1` actually indicates volume is unhealth maybe weird.

Reply:
VolumeStatsHealthStatusKey may also needs to change

Reply:
I can change comment part. But, I am a fan to change metrics key name from `volume_stats_health_status` to `volume_stats_abnormal_health_status`. There is a label inside the metrics to indicate the volume stat is normal or abnormal. I don't see there is any difference between them. Put "abnormal" or "normal" as the part of metrics name will cause a new metrics when we want to have normal one in the future.

And the KEP also indicates this metrics name like that: https://github.com/kubernetes/enhancements/pull/2900/files?short_path=15ee612#diff-15ee61212fda1ec3f574c71df1ad7c03847a33089b9d68740aa68bc598797822

Reply:
I think we should keep the metrics key name and use the label to indicate if it is abnormal or not.